image/image_test.go

```diff
@@ -54,6 +55,28 @@ func TestMarshalKeyOrder(t *testing.T) {
 	}
 }
 
+func TestImageWithAllMethods(t *testing.T) {
+	var cid = "50a16564e727"
+	var raw = []byte("279533a6d3a9")
+	config := &container.Config{}
```

Comment:
Maybe there should be _something_ in this config so that we're not comparing an empty config